src/Phoenix/PhoenixClient.ts

```diff
@@ -71,28 +81,53 @@ export class PhoenixClient {
   private async getContainerStatusAsync(containerData: IContainerData): Promise<ContainerInfo> {
     try {
       const runContainerStatusAsync = async () => {
+        TelemetryProcessor.traceStart(Action.PhoenixHeartBeat, {
+          dataExplorerArea: Areas.Notebook,
```

Comment:
why do we need ths telemetry..
for 1 user - this will be every few sec/minutes.. and will beso noisy.. do we gain benefit by having this?

Reply:
Added this if we want to know the rate of heart beat success